storage/storage.go

```diff
@@ -355,13 +355,6 @@ type TupleBackend interface {
 		filter ReadStartingWithUserFilter,
 	) (TupleIterator, error)
 
-	// ReadByStore reads the tuples associated with `store`.
```

Comment:
Can you update the documentation in line 300?

Reply:
Good call. Thank you! Updated.